main.js

```diff
@@ -0,0 +1,111 @@
+// Ex 1: Thuế và lương ròng
+const getSalery = (salery) => {
+    if(salery >= 15000000){
+        console.log('Thuế thu nhập =',salery*30/100);
+        console.log('Lương ròng = ',salery*70/100);
+    }
+    else if (salery < 15000000 && salery > 7000000) {
+        console.log('Thuế thu nhập =',salery*20/100);
+        console.log('Lương ròng =',salery*80/100);
+    }
+    else if (salery < 7000000 && salery > 0) {
+        console.log('Thuế thu nhập = ',salery*10/100);
+        console.log('Lương ròng = ',salery*90/100);
+    }
+}
+
+// Ex 2: tuổi vào lớp 10 của học sinh
+function ageClass(tuoi) {
+    if(tuoi == 16) return 'Đủ điều kiện vào học lớp 10';
+    else if (tuoi < 16 || tuoi > 16) return 'Không đủ điều kiện vào học lớp 10';
+}
+
+// Bài 3: lớn hay nhỏ hơn 100
+const compareVs100 = (number) => {
+    if(number > 100) {
+        return 'Số lớn hơn 100';
+    } else if (number < 100) {
+        return 'Số nhỏ hơn 100';
+    } else {
+        return 'Số bằng 100';
+    }
+}
+
+// Bài 4 Tìm số lớn nhất trong 3 số 
+const maxnum = (a,b,c) => {
```

Comment:
```suggestion
const getMaxNum = (a, b, c) => {
```
hoặc
```suggestion
const findMaxNumber = (a, b, c) => {
```
nhìn kỹ cách đặt tên và đặt space